CodeGeneration/Sources/SyntaxSupport/Classification.swift.gyb

```diff
@@ -34,6 +34,25 @@ public class SyntaxClassification {
   }
 }
 
+public class ChildClassification {
+  public let parent: Node
+  public let childIndex: Int
+  public let isToken: Bool
+  public let classification: SyntaxClassification?
+  public let force: Bool
+  public let swiftForce: String
```

Comment:
If you want to have this property, I think it should be a computed property. But I would prefer to just do the conversion to string on the call side.